config/default.json

```diff
@@ -24,9 +24,8 @@
     "gaTrackingId": ""
   },
   "fees": {
-    "url": "http://localhost:8182",
-    "issueFeeCode": "onlinefees",
-    "hearingFeeCode": "hearingfees"
+    "url": "https://dev-proxy.fees-register.reform.hmcts.net",
```

Comment:
reverted to use local, as has a branch that contains fix for fees-api docker image to use 
 locally.